docs/topics/testing/tools.txt

```diff
@@ -153,23 +158,33 @@ Use the ``django.test.Client`` class to make requests.
 
             /customers/details/?name=fred&age=7
 
-        The ``extra`` keyword arguments parameter can be used to specify
-        headers to be sent in the request. For example::
+        The ``headers`` parameter can be used to specify headers to be sent in
+        the request. For example::
+
+            >>> c = Client()
+            >>> c.get('/customers/details/', {'name': 'fred', 'age': 7},
+            ...       headers={'Accept': 'application/json'})
+
+        Arbitrary keyword arguments set WSGI `environ variables`_. For example,
+        headers are set with the ``HTTP_`` prefix::
+
 
             >>> c = Client()
             >>> c.get('/customers/details/', {'name': 'fred', 'age': 7},
             ...       HTTP_ACCEPT='application/json')
 
+        .. _environ variables: https://peps.python.org/pep-3333/#environ-variables
+
         ...will send the HTTP header ``HTTP_ACCEPT`` to the details view, which
         is a good way to test code paths that use the
         :meth:`django.http.HttpRequest.accepts()` method.
 
         .. admonition:: CGI specification
 
-            The headers sent via ``**extra`` should follow CGI_ specification.
-            For example, emulating a different "Host" header as sent in the
-            HTTP request from the browser to the server should be passed
-            as ``HTTP_HOST``.
+            The headers sent via ``headers`` and ``**extra`` should follow CGI_
+            specification. For example, emulating a different "Host" header as
+            sent in the HTTP request from the browser to the server should be
+            passed as ``HTTP_HOST``.
 
             .. _CGI: https://www.w3.org/CGI/
```

Comment:
I think we can remove this CGI admonition since we now link to WSGI environ vars, which are based on CGI, and it will be irrelevant for using `headers`

It's also not true that `headers` should follow CGI